modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsChunksViewDisplayMapping.php

```diff
@@ -7,7 +7,23 @@
 use Drupal\migrate\Row;
 
 /**
- * Process Plugin to map view and display for Chunks View paragraphs.
+ * Process Plugin to map view and display for Chunks View paragraphs. This
+ * plugin expects a source value which is an associative containing the keys
+ * "vname" and "vargs".
+ *
+ * Available configuration keys
+ * - N/A
+ *
+ * Examples:
+ *
+ * Consider a paragraph item migration, where you want to preserve the view
+ * display mapping.
+ * @code
+ * process:
+ *   field_az_view_reference:
+ *     plugin: paragraphs_chunks_view_display_mapping
```

Comment:
I wonder if it would be possible to come up with a more clear name/documentation for this one. Without actually reading the code, I'm still not sure what use cases this class handles. To be fair, I have not worked with the paragraph module very much in the past.

Reply:
I have another (draft) PR open related to this plugin that does a lot of cleanup:
https://github.com/az-digital/az_quickstart/pull/1109